pkg/parent/parent.go

```diff
@@ -244,6 +244,10 @@ func listenServeAPI(socketPath string, backend *router.Backend) (apiCloser, erro
 	r := mux.NewRouter()
 	router.AddRoutes(r, backend)
 	srv := &http.Server{Handler: r}
+	err := syscall.Unlink(socketPath)
+	if err != nil {
+		return nil, err
+	}
```

Comment:
Thanks, but CI seems failing

```
 rootlesskit --net=slirp4netns --mtu=1500 iperf3 -t 60 -c 10.0.2.2
[rootlesskit:parent] error: no such file or directory
```

Could you use `os.RemoveAll` instead of `syscall.Unlink`? https://golang.org/pkg/os/#RemoveAll

Also, please sign your commit with your full name?  (`git commit -a -s --amend`)